From 5575038365160e6bbdc242ab9fcad9ba1abd50ab Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <Michal@Hrusecky.net>
Date: Tue, 16 Jan 2018 16:09:14 +0100
Subject: [PATCH] ncurses: Provide full host build

Some packages might need full ncurses to do host build before doing cross
build. Like MariaDB.

Signed-off-by: Michal Hrusecky <Michal@Hrusecky.net>
---
 package/libs/ncurses/Makefile | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/package/libs/ncurses/Makefile b/package/libs/ncurses/Makefile
index 94c9688..a687fe1 100644
--- a/package/libs/ncurses/Makefile
+++ b/package/libs/ncurses/Makefile
@@ -84,7 +84,7 @@ HOST_CONFIGURE_ARGS += \
 	--without-manpages \
 	--without-profile \
 	--without-tests \
-	--without-curses-h
+	--prefix=$(STAGING_DIR_HOSTPKG)
 
 
 ifeq ($(HOST_OS),FreeBSD)
@@ -158,14 +158,11 @@ define Build/InstallDev
 endef
 
 define Host/Compile
-	$(MAKE) -C $(HOST_BUILD_DIR) libs
-	$(MAKE) -C $(HOST_BUILD_DIR)/progs tic
+	$(MAKE) -C $(HOST_BUILD_DIR)
 endef
 
 define Host/Install
-	$(INSTALL_BIN) $(HOST_BUILD_DIR)/progs/tic $(STAGING_DIR_HOST)/bin/tic
-	$(INSTALL_DIR) $(STAGING_DIR_HOSTPKG)/lib/
-	$(INSTALL_DATA) $(HOST_BUILD_DIR)/lib/libncurses.a $(STAGING_DIR_HOSTPKG)/lib/
+	$(MAKE) -C $(HOST_BUILD_DIR) install
 endef
 
 $(eval $(call HostBuild))
-- 
2.18.0

